pyproject.toml

```diff
@@ -42,7 +42,7 @@ importlib-metadata = { version = ">=4.4", python = "<3.10" }
 installer = "^0.7.0"
 keyring = "^24.3.1"
 # packaging uses calver, so version is unclamped
-packaging = ">=23.1"
+packaging = ">=24.0"
```

Comment:
Is there a specific reason for this? (Nothing in the changelog caught my eye.)

Reply:
Probably overzealous of me. I can revert that if you want.

Reply:
since poetry-core vendors `packaging` and puts its vendored copy at the front of the `sys.path`, it makes no difference what version poetry asks for...